onnxruntime/core/session/inference_session.cc

```diff
@@ -1024,16 +1024,30 @@ Status InferenceSession::LoadOrtModelWithLoader(std::function<Status()> load_ort
   const auto* fbs_ort_model_version = fbs_session->ort_version();
   ORT_RETURN_IF(fbs_ort_model_version == nullptr, "Serialized version info is null. Invalid ORT format model.");
 
+  auto model_version = std::stoi(std::string(fbs_ort_model_version->str()));
```

Comment:
nit: flatbuffers::String::str() returns std::string
```suggestion
  auto model_version = std::stoi(fbs_ort_model_version->str());
```